src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -21,6 +21,7 @@
 metric to be for edges, crash, features, or units. Currently based on new
 edges."""
 
+from bot.fuzzers import strategy
```

Comment:
src/python/bot isn't accessible from app engine. We can get rid of that restriction but it makes our App Engine deployments take a bit longer. Maybe we will have to move this to somewhere like src/python/fuzzing after all.

Reply:
(Note: see src/appengine/skip_files.yaml for context on the previous comment)